packages/tasks/src/tasks/image-to-3d/about.md

```diff
@@ -0,0 +1,50 @@
+## Use Cases
+
+Image-to-3D models can be used in a wide variety applications that require 3D, such as games, animation, design, architecture, engineering, marketing, and more.
+
+### Meshes
+
+Meshes are the standard representation of 3D in industry.
```

Comment:
Would be nice to put an image of how a mesh looks like and explain what a mesh generation model takes input and outputs output